src/components/widgets-settings/forecast-payroll/forecast-payroll.directive.coffee

```diff
@@ -0,0 +1,50 @@
+module = angular.module('impac.components.widgets-settings.forecast-payroll',[])
+module.controller('SettingForecastPayrollCtrl', ($scope, $log, ImpacDashboardsSvc, ImpacMainSvc, ImpacWidgetsSvc) ->
+
+  w = $scope.parentWidget
+  w.selectedForecastPayroll = false
+
+  $scope.toggleForecastPayroll = ->
+    w.selectedForecastPayroll = !w.selectedForecastPayroll
+
+  $scope.isToggleForecastPayroll = ->
+    return w.selectedForecastPayroll
+
+  # What will be passed to parentWidget
+  setting = {}
+  setting.key = "forecast_payroll"
+  setting.isInitialized = false
+
+  # initialization of selected organizations
+  setting.initialize = ->
+    ImpacDashboardsSvc.load().then(
+      (config) ->
+        currentDashboard = config.currentDashboard
+        return unless w.metadata? && w.metadata.forecast_payroll?
+        w.selectedForecastPayroll = w.metadata.forecast_payroll
```

Comment:
Is there a reason you're attaching `selectedForecastPayroll` onto the widget object? I know impac-angular does this across the framework, but that is old architecture and it's not really a good practice. Based on what I can see here, I would just use `$scope.selectedForecastPayroll`.